### PR TITLE
Heartbeat interval configurable

### PR DESCRIPTION
**Pull request by engelmi**
This PR solves #133 and makes the heartbeat interval configurable via CLI option as well as configuration file.

### CODE REVIEW

**Review by sdunnagan — APPROVED** [review]

File: config/agent/agent.conf [hunk 1]
```diff
@@ -26,6 +26,10 @@ ManagerHost=
 # CLI options.
 #ManagerAddress=
 
+#
+# Defines the interval between two heartbeat signals sent to hirte in milliseconds. 
+#HeartbeatInterval=
```
Comment (mwperina) [inline config/agent/agent.conf:31]: Does the option have a default value? If so, please add it here

File: config/agent/hirte-default.conf [hunk 2]
```diff
@@ -22,6 +22,10 @@ ManagerPort=842
 # CLI options.
 ManagerAddress=
 
+#
+# Defines the interval between two heartbeat signals sent to hirte in milliseconds. 
+HeartbeatInterval=
```
Comment (mwperina) [inline config/agent/hirte-default.conf:27]: Same as above

File: doc/man/hirte-agent.1.md [hunk 3]
```diff
@@ -39,6 +39,10 @@ Overrides any setting of `ManagerHost` or `ManagerPort` defined in the configura
 
 The unique name of this `hirte-agent` used for registering at `hirte`. This option will overwrite the port defined in the configuration file.
 
+#### **--interval**, **-i**
+
+The interval between two heartbeat signals sent to hirte in milliseconds. This option will overwrite the heartbeat interval defined in the configuration file. 
+
```
Comment (mwperina) [inline doc/man/hirte-agent.1.md:45]: Same as above

File: doc/man/hirte-agent.conf.5.md [hunk 4]
```diff
@@ -40,6 +39,10 @@ value, it's mandatory to set this option for each hirte-agent.
 The port on which `hirte` is listening for connection request and the `hirte-agent` is connecting to. By default port
 `842` is used.
 
+#### **HeartbeatInterval** (long)
+
+The interval between two heartbeat signals sent to hirte in milliseconds. This option will overwrite the heartbeat interval defined in the configuration file. 
+
```
Comment (mwperina) [inline doc/man/hirte-agent.conf.5.md:45]: Same as above

**Pull request merged**